frontend/src/lib/components/InsightLabel/index.tsx

```diff
@@ -74,6 +75,13 @@ function MathTag({ math, mathProperty, mathGroupTypeIndex }: MathTagProps): JSX.
             </>
         )
     }
+    if (math === 'hogql') {
+        return (
+            <Tag title={String(mathHogQL)}>
+                {String(mathHogQL).length > 10 ? `${mathHogQL?.substring(0, 10)}...` : String(mathHogQL ?? '')}
```

Comment:
The 10 character limit is a bit frustrating, having just played around with this. It's short enough that even something extremely trivial like `sum(property.amount)` gets cut off, which makes those legends effectively useless. Can we relax this limit? Ideally relying on CSS

Reply:
Makes sense! How about `max-w-60 text-ellipsis overflow-hidden`?

<img width="1237" alt="image" src="https://github.com/PostHog/posthog/assets/53387/ce9a8a59-3755-4d22-9947-68934356329c">